packages/rocketchat-lib/server/models/Subscriptions.js

```diff
@@ -12,7 +12,7 @@ class ModelSubscriptions extends RocketChat.models._Base {
 		this.tryEnsureIndex({ 'unread': 1 });
 		this.tryEnsureIndex({ 'ts': 1 });
 		this.tryEnsureIndex({ 'ls': 1 });
-		this.tryEnsureIndex({ 'audioNotification': 1 }, { sparse: 1 });
+		this.tryEnsureIndex({ 'audioNotifications': 1 }, { sparse: 1 });
```

Comment:
I changed audioNotification to 'audioNotifications' so that all will be of the same form, like 'desktopNotifications', 'mobileNotifications'.